wazo_dird/plugins/phonebook_backend/plugin.py

```diff
@@ -88,8 +88,6 @@ def load(self, dependencies: Dependencies):
         searched_columns = config.get('searched_columns')
         first_matched_columns = config.get('first_matched_columns')
 
-        self._crud = database.PhonebookCRUD(Session)
```

Comment:
How `_crud` is initalized ?

Reply:
Good question, the line was removed by mistake as `_crud` is still used

Reply:
Worryingly, that was not caught by tests as the tests use patch mocking on the `_crud` when the implementation would use it.